hawc/apps/study/templates/study/study_clone.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'assessment-rooted.html' %}
+
+{% block content %}
+  {% crispy form %}
+  <br>
+  <h4>Select studies*</h4>
+  {% for assessment_id, studies in study_data.items %}
+    <table style="display: none;" id="{{assessment_id}}" class='table table-sm table-striped studies'>
+      {% bs4_colgroup '40,20,20,20' %}
+      {% bs4_thead 'Short citation,Bioassay,Epi,Risk of Bias' %}
+      <tbody>
+        {% for study in studies %}
+          <tr>
+            <td>
+              <input type="checkbox" id="study_{{study.id}}" value="{{study.id}}">
+              <label style="font-weight: normal;" for="study_{{study.id}}">&nbsp;{{study}}</label>
+            </td>
+            <td>
+              {% if study.bioassay %}
+                <input type="checkbox" value="{{study.bioassay}}">
+              {% else %}
+                <input disabled type="checkbox" value="{{study.bioassay}}">
+              {% endif %}
+            </td>
+            <td>
+              {% if study.epi and assessment.epi_version == 2 and study.assessment.epi_version == 2 %}
+                <input type="checkbox" value="{{study.epi}}">
+              {% else %}
+                <input disabled type="checkbox" value="{{study.epi}}">
+              {% endif %}
+            </td>
+            <td>
+              {% if study.rob %}
+                <input type="checkbox" name="rob" value="{{study.rob}}">
+              {% else %}
+                <input disabled name="rob" type="checkbox" value="{{study.rob}}">
+              {% endif %}
+            </td>
+          </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+  {% endfor %}
+  <br>
+  <h4>Map metrics*</h4>
```

Comment:
to map the metrics, I think you'll need to select the src assessment at a prior stage, we dont want to map the metrics for all possible assessments here. It may make more sense to make two pages (or an htmx command) where you'd select your src assessment first, and then this populates with just the info for the src assessment.

For now, you could just update the url to do something like `assessment/<int:pk>/clone-from/<int:src_pk>/` and then if someone visited `assessment/1/clone-from/2/`, the src would be assessment 2 and dst would be assessment 1.